docs/smtchecker.rst

```diff
@@ -697,7 +697,7 @@ storage for ``address`` variables, therefore if ``B.a`` had type ``address``
 the encoding would assume that its storage does not change in between
 transactions to ``B``.
 
-   .. code-block:: solidity
+.. code-block:: solidity
```

Comment:
I'm not sure why this error [here](https://app.circleci.com/pipelines/github/ethereum/solidity/30437/workflows/0267a43b-d122-49db-a46b-de8dae252d1a/jobs/1350968) started to happen after your change (i.e. removing the leading space from the code-block). But you can fix it by adding `// SPDX-License-Identifier: GPL-3.0` before the `pragma` in this example. But I'm wondering if there is any other example in the docs that is not being compiled.

Reply:
> I'm not sure why this error [here](https://app.circleci.com/pipelines/github/ethereum/solidity/30437/workflows/0267a43b-d122-49db-a46b-de8dae252d1a/jobs/1350968) started to happen after your change

That's how our regex is constructed. It requires no whitespace there: https://github.com/ethereum/solidity/blob/27faacb72c2fd6627077821a377fcdb454c91022/scripts/isolate_tests.py#L39-L49

I would actually expect Sphinx not to detect it as a proper code block (but instead either error out or treat it as the content of an idented block). It it's valid, we should adjust the regex. If it's not, perhaps the script should warn/fail when it finds such blocks.

Reply:
Ah right. But this would be done in a separate PR. For this one, it should be sufficient to just add the SPDX-License identifier, right?

Reply:
Yeah, sure. Just pointing out why it happens.